Monika-After-Story/MonikaModDev
# New music menu

Gonna refactor the entire music menu.

### TODO:
- [ ] show current track
- [x] show position of the current track
- [x] allow changing position of the current track
- [ ] allow pause/unpause
- [ ] add volume slider

### Changes:
- refactored the music menu, but exactly:
  - we show the current track now.
  - added the ability to pause music.
  - the music menu now cuts off songs names that contain more than 27 characters and shows a tooltip instead.
  - user can see and change the position of the current track.
- fixed the issue with navigation buttons (and other buttons that inherit the style) having hitboxes larger than the text inside them.

### Testing:
- verify the main menu and music menu look good, and their buttons have appropriate hitboxes
- verify the music menu looks good even if you have songs with long names
- verify the new tooltip works
